mtools/mlaunch/mlaunch.py

```diff
@@ -56,6 +56,9 @@ class MongoConnection(Connection):
     """
```

Comment:
Can you update this docstring as well as remove the outdated pymongo 2.X era imports above?

Reply:
Done, thanks!